code/orders-api-rest/api/openapi-rest.yml

```
@@ -2231,3 +2266,35 @@ components:
+          example: John
+        lastName:
+          type: string
+          description: User's last name. Must be between 1 and 50 characters.
+          minLength: 1
+          maxLength: 50
+          example: Doe
+        phone:
+          type: string
+          description: |
+            User's phone number in international or local format.
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)